### DISCUSSION TITLE
Look and feel customization

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore.Commerce#457] [Q&A] [answered]
Hi,
How can I customize the look and feel of all contents from [OrchardCore.Commerce](https://github.com/OrchardCMS/OrchardCore.Commerce)?

![](https://commerce.orchardcore.net/en/latest/assets/images/create-webshop/step-4/product-list.png)

![](https://commerce.orchardcore.net/en/latest/assets/images/create-webshop/price-variants-products/view-variants-product.png)

**Comment by sarahelsaig** [comment, marked as answer]
Out of the box the demo setup recipe uses [Orchard Core's built-in TheBlogTheme](https://github.com/OrchardCMS/OrchardCore.Commerce/blob/main/src/Modules/OrchardCore.Commerce/Recipes/OrchardCore.Commerce.Development.Setup.recipe.json#L81). (We don't have a dedicated commerce theme yet, see issue #300.) OrchardCore.Commerce has very little custom styling. It relies on Bootstrap for basic structure and components. So the simplest way to customize the site is to switch to a different Bootstrap compatible Orchard Core theme in Admin → Design → Themes.

When you [create your custom theme](https://docs.orchardcore.net/en/latest/getting-started/theme/) you can customize the site's general look and feel using CSS, for example you can [customize of Bootstrap CSS variables](https://getbootstrap.com/docs/5.0/customize/css-variables/) to adjust the site's overall color theme. You can also customize individual OrchardCore.Commerce cshtml files by [overriding them](https://www.youtube.com/watch?v=bwFH-C18rrA) in your theme. Also see the Orchard Core documentation [here](https://docs.orchardcore.net/en/latest/reference/modules/Themes/). These are all stock Orchard Core features, so if you need further support you can ask in the [Orchard Core discussions](https://github.com/OrchardCMS/OrchardCore/discussions) or the orchard-core channel on [Discord](https://orchardcore.net/discord).
